simple-user-login-system/app/__init__.py

```diff
@@ -0,0 +1,14 @@
+from flask import Flask
+import os
+
+
+def create_app():
+    app = Flask(__name__)
+    if os.getenv("ENV")=="development":
+        app.config.from_object("config.DevelopmentConfig")

```

Comment:
Where does the `config.DevelopmentConfig` object come from?

Reply:
my config.DevelopmentConfig object is comming from my config.py file which i have acctually added it to git ignore.

Reply:
This is actually how my configuration looks like in config.py file

class Config(object):

    DEBUG = False
    TESTING = False
    SECRET_KEY = ''
    HOST = 'localhost'
    DB = 'production_db'
    USER = 'admin'
    PASSWORD = 'password'
    UPLOADS = ''
    SESSION_COOKIE_SECURE = True

class ProductionConfig(Config):
    pass

class DevelopmentConfig(Config):
    DEBUG = True    
    HOST = "localhost"
    DB = "login"
    USER = "root"
    PASSWORD = "Tyondo@98" 
    SESSION_COOKIE_SECURE = False

class TestingConfig(Config):
    TESTING = True  
    HOST = "localhost"
    DB = "logintest"
    USER = "root"
    PASSWORD = 'Tyondo@98'
    SESSION_COOKIE_SECURE = False

Reply:
so nikiwa kwa development environment natumia login database, nikiswitch to testing environment natumia logintest database.

Reply:
Done.